File: ExercicioComparacaoLinguagens/atividade_encapsulamento/php/dtos/Pessoa.php
<?php
class Pessoa {
    private $name;
    private $idade;
    private $cpf;
    private $profissao;

    public function __construct($name, $idade, $cpf) {
        $this->name = $name;
        $this->idade = $idade;
        $this->cpf = $cpf;
    }

    // Getters e Setters
    public function getName() {
        return $this->name;
    }

    public function setName($name) {
        $this->name = $name;
    }

    public function getIdade() {
        return $this->idade;
    }

    public function setIdade($idade) {
        $this->idade = $idade;
    }

    public function getCpf() {
        return $this->cpf;
    }

    public function setCpf($cpf) {
        $this->cpf = $cpf;
    }

    public function getProfissao() {
        return $this->profissao;
    }

    public function setProfissao($profissao) {
        $this->profissao = $profissao;
    }

    public function __toString() {
        return "Pessoa [name=" . $this->name . ", idade=" . $this->idade . ", cpf=" . $this->cpf . ", profissao=" . $this->profissao . "]";
    }
}
?>

File: ExercicioComparacaoLinguagens/atividade_encapsulamento/java/encapsulamento/src/dtos/Pessoa.java
package dtos;

public class Pessoa {
	// atributos private acessiveis apenas para classe pessoa 
	private String name;
	private int idade;
	private int cpf;
	protected Profissao profissao ; 
	
	public Pessoa() {}
	public Pessoa(String name, int idade, int cpf) {
		super();
		this.name = name;
		this.idade = idade;
		this.cpf = cpf;
	}
	//metodos get e set para acessar de outra classe 
	public String getName() {
		return name;
	}
	public void setName(String name) {
		this.name = name;
	}
	public int getIdade() {
		return idade;
	}
	public void setIdade(int idade) {
		this.idade = idade;
	}
	public int getCpf() {
		return cpf;
	}
	public void setCpf(int cpf) {
		this.cpf = cpf;
	}
	public Profissao getProfissao() {
		return profissao;
	}
	public void setProfissao(Profissao profissao) {
		this.profissao = profissao;
	}
	@Override
	public String toString() {
		return "Pessoa [name=" + name + ", idade=" + idade + ", cpf=" + cpf + ", profissao=" + profissao + "]";
	}
	
 
	
}
